stages/org.osbuild.anaconda

```diff
@@ -33,24 +56,31 @@ CONFIG = """
 # osbuild customizations
 
 [Anaconda]
-# List of enabled Anaconda DBus modules
-kickstart_modules =
 """
 
 
 def main(tree, options):
-    modules = options["kickstart-modules"]
     product_dir = os.path.join(tree, "etc/anaconda/conf.d")
     os.makedirs(product_dir, exist_ok=True)
 
-    with open(os.path.join(product_dir, "90-osbuild.conf"), "w", encoding="utf8") as f:
+    with open(
+        os.path.join(product_dir, "90-osbuild.conf"), "w", encoding="utf8"
+    ) as f:
         f.write(CONFIG)
-        for m in modules:
-            f.write(f"    {m}\n")
+
+        for grouping in [
+            "kickstart-modules",
+            "activatable-modules",
+            "forbidden-modules",
+            "optional-modules",
+        ]:
+            if grouping in options:
+                f.write(grouping.replace("-", "_") + "\n")
```

Comment:
I'm sorry for not seeing this earlier, but I think this line misses putting ` = ` before the newline.

Reply:
Yes, this needs a `=` for the group name.

Reply:
Ack, done.